Fix OpenAI client usage example for embeddings

ZahraDehghani99 opened the pull request:
### Title

**docs(tei): fix OpenAI client usage example for embeddings**

---

### PR Description

This PR updates the documentation for using the OpenAI Python client with TEI’s OpenAI-compatible API.
The previous example incorrectly set the `base_url` to `/v1/embeddings`, which caused errors.
It also did not mention that the OpenAI client requires an `api_key` argument (even if unused by TEI).

**Changes made:**

* Corrected the `base_url` to `http://localhost:8080/v1`
* Added `api_key="fake"` to satisfy the OpenAI client requirements
* Updated the example to print `response.data[0].embedding` for clarity

**Motivation:**
Improve usability of the TEI docs by providing a working out-of-the-box example for developers who want to use the OpenAI client with TEI.

**Fixes:**
N/A (documentation improvement)

---

### Before submitting

* [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
* [x] Did you read the [contributor guideline](https://github.com/huggingface/text-embeddings-inference/blob/main/CONTRIBUTING.md)?
* [x] Did you make sure to update the documentation with your changes?
* [x] Tests not applicable (docs-only change).

---

### Who can review?

Tagging maintainer for review:
@Narsil

alvarobartt commented on docs/source/en/quick_tour.md:
````diff
@@ -77,21 +77,22 @@ print(len(embedding[0]))
 ```
 
 #### OpenAI
+The server is availabe at : localhost:8080/v1/embeddings
````
```suggestion
```

alvarobartt commented on docs/source/en/quick_tour.md:
```diff
@@ -77,21 +77,22 @@ print(len(embedding[0]))
 from openai import OpenAI
 
-client = OpenAI(base_url="http://localhost:8080/v1/embeddings")
+client = OpenAI(base_url="http://localhost:8080/v1", api_key= "fake")
```
```suggestion
client = OpenAI(base_url="http://localhost:8080/v1", api_key= "-")
```

alvarobartt commented on docs/source/en/quick_tour.md:
````diff
@@ -77,21 +77,22 @@ print(len(embedding[0]))
-print(response)
+print(response.data[0].embedding)
 ```
 
````
Maybe also add `curl` example? Note that the closing backticks are missing another one, but GitHub won't let me add that and still be compliant with the suggestion format, so you'll need to add it afterwards 🤗 

```suggestion

Alternatively, you can also send the request with cURL as follows:

```bash
curl http://localhost/v1/embeddings \
  -H "Content-Type: application/json" \
  -d '{
    "input": "What is Deep Learning?",
    "model": "text-embeddings-inference",
    "encoding_format": "float"
  }'
``

```

alvarobartt commented on docs/source/en/quick_tour.md:
```diff
@@ -77,21 +77,22 @@ print(len(embedding[0]))
+client = OpenAI(base_url="http://localhost:8080/v1", api_key= "fake")
 
 response = client.embeddings.create(
   model="tei",
   input="What is deep learning?"
```
```suggestion
    model="text-embeddings-inference",
    input="What is Deep Learning?",
```

alvarobartt commented on docs/source/en/quick_tour.md:
```diff
@@ -77,21 +77,22 @@ print(len(embedding[0]))
 #### OpenAI
+The server is availabe at : localhost:8080/v1/embeddings
 
 You can install it via pip as `pip install --upgrade openai`, and then run:
```
```suggestion
To send requests to the [OpenAI Embeddings API](https://platform.openai.com/docs/api-reference/embeddings/create) exposed on Text Embeddings Inference (TEI) with the OpenAI Python SDK, you can install it as `pip install --upgrade openai`, and then run the following snippet:
```

ZahraDehghani99 commented:
> Hey @ZahraDehghani99 thanks a lot for the PR! I've included some extra comments, but other than that LGTM 🤗

Thank you so much @alvarobartt for all your helpful suggestions! I’ve applied all of them and cleaned up the examples, including fixing the backticks and removing the accidentally added duplicate from GitHub UI. I also added the port number in the curl request so it works correctly. Please let me know if there’s anything else I should adjust 😊

alvarobartt commented:
Thanks @ZahraDehghani99, merging to main now 🔥
 
<img width="1002" height="740" alt="image" src="https://github.com/user-attachments/assets/0016d091-4d6e-4cd1-a24a-3e02cab7b69a" />
